design-tokens/src/token_export.ts

```diff
@@ -4,6 +4,16 @@ import { ApiGetLocalVariablesResponse, Variable } from './figma_api.js';
 import { Token, TokensFile } from './token_types.js';
 import { access } from './utils.js';
 
+/**
+ * Supported color interaction states
+ */
+const interactions = ['hover', 'focus', 'active'];
```

Comment:
FYI -- having to define these interactions/prominences/exceptions here is not ideal, but I feel it's sufficient for the time being. Something we can come back to refine in future.